<launch>
 
  <include file="$(find temoto)/launch/pocketsphinx.launch" />
 
 <node name="leap_motion" pkg="leap_motion_controller" type="leap_motion" />

  <node name="rviz_visual" pkg="temoto" type="rviz_visual" />
  
  <node name="navigate_robot" pkg="temoto" type="navigate_robot" />

  <node name="listen_operator_talk" pkg="temoto" type="listen_operator_talk" />
  <node name="async_tf_broadcaster" pkg="temoto" type="async_tf_broadcaster" />


  <!-- bringup proper rviz config -->
  
</launch>